src/main/java/teammates/ui/request/FeedbackSessionUpdateRequest.java

```diff
@@ -1,8 +1,54 @@
 package teammates.ui.request;
 
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * The update request of a feedback session.
  */
 public class FeedbackSessionUpdateRequest extends FeedbackSessionBasicRequest {
 
```

Comment:
I'm not entirely sure, would it be better to make a separate endpoint to update deadlines from feedback session? Currently we only want to update these 3 fields

Reply:
Would it feel RESTful still? I think this is equivalent to using PATCH to some extent (only patch up the deadline fields).

In this particular case, we are updating deadlines separately from the session itself. So I am not shutting this idea entirely. @samuelfangjw any thoughts on this?

Reply:
Resolved via conversation on slack.

Reply:
Following up on this, then would it be better to make the deadline fields optional in the update request? Not all feedback session update requests would touch on the deadlines

Reply:
Answered by Samuel here: https://github.com/TEAMMATES/teammates/pull/11603#discussion_r830816498. It'll be less complicated without the optional fields